src/lib/mathlib/math/Functions.hpp

```diff
@@ -41,15 +41,11 @@
 
 #include "Limits.hpp"
 
-namespace math
-{
+#include <matrix/matrix/math.hpp>
+using matrix::sign;
```

Comment:
I'm a little hesitant to put `sign` into the global namespace, since it may collide with a C function in a library an there is no way to fix it on the C side.

Reply:
Oh you're right that obviously gets through the include chain. Let me fix that.

Reply:
Fixed, thanks for catching.